Jenkinsfile.internal

```diff
@@ -31,7 +31,7 @@ pipeline {
         pipeline_report_dir = "report2TG/build-$env.BUILD_NUMBER"
         PRODUCT_NAME = "ecutest-plugin"
         PRODUCT_VERSION = readMavenPom().getVersion()
-        TEST_LEVEL = "Unit Tests"
+        TEST_LEVEL = "unit"
```

Comment:
GIT_URL and GIT_COMMIT is missing

Reply:
obsolete - the test.guide upload already contains both keys. But... which tool/script defines it? I cannot find it.